src/patternfly/components/Select/select-checkbox-groups.hbs

```diff
@@ -0,0 +1,48 @@
+  {{#if select-multi--attribute}}
+    {{{select-multi-attribute}}}
+  {{/if}}>
+  {{#> form}}
+    {{#> form-group form-group--modifier="pf-c-form__fieldset" form-group--attribute=(concat 'aria-labelledby="' id '-label"')}}
+      <div class="pf-c-select__menu-group">
+        <div class="pf-c-select__menu-group-title">Status</div>
+        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-running"') check--modifier="pf-c-select__menu-item"}}
+          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-running" name="running"')}}{{/check-input}}
+          {{#> check-label check-label--type="span"}}Running{{/check-label}}
+        {{/check}}
+        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-stopped"') check--modifier="pf-c-select__menu-item"}}
+          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-stopped" name="stopped" checked')}}{{/check-input}}
+          {{#> check-label check-label--type="span"}}Stopped{{/check-label}}
+        {{/check}}
+        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-down"') check--modifier="pf-c-select__menu-item"}}
+          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-down" name="down" checked')}}{{/check-input}}
+          {{#> check-label check-label--type="span"}}Down{{/check-label}}
+        {{/check}}
+        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-degraded"') check--modifier="pf-c-select__menu-item"}}
+          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-degraded" name="degraded"')}}{{/check-input}}
+          {{#> check-label check-label--type="span"}}Degraded{{/check-label}}
+        {{/check}}
+        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-needsMaintenance"') check--modifier="pf-c-select__menu-item"}}
+          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-needsMaintenance" name="needsMaintenance" checked')}}{{/check-input}}
+          {{#> check-label check-label--type="span"}}Needs Maintenance{{/check-label}}
+        {{/check}}
+      </div>
+      <div class="pf-c-select__menu-group">
+        <div class="pf-c-select__menu-group-title">Vendor</div>
+        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-dell"') check--modifier="pf-c-select__menu-item"}}
+          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-dell" name="dell"')}}{{/check-input}}
+          {{#> check-label check-label--type="span"}}Dell{{/check-label}}
+        {{/check}}
+        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-samsung"') check--modifier="pf-c-select__menu-item pf-m-disabled"}}
+          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-samsung" name="samsung" aria-disabled="true" disabled')}}{{/check-input}}
```

Comment:
you can remove `aria-disabled` on any form element that supports the `disabled` attribute. that's from https://github.com/patternfly/patternfly-next/issues/1209

Reply:
Ugh, that got lost in refactoring for the new checkbox.